### ISSUE TITLE
per_image_standardization is not recognized

### ISSUE THREAD

**Issue by hamidb** [tensorflow/tensorflow#5774]
Hi, 
I installed Tensorflow with Python3 using the pip3 installation instructions from Tensorflow official site.
The version I installed is `0.11.0rc2` and I tried to use `per_image_standardization` tensor for normalization purposes. However it turned out that is not recognized, giving the following error:
```
AttributeError: module 'tensorflow.python.ops.image_ops' has no attribute 'per_image_standardization'
```
Has this error ever observed by someone else?

**Comment by prb12** [comment]
I believe it was recently renamed from `per_image_whitening` in this commit: https://github.com/tensorflow/tensorflow/commit/3adb3fd05f816bb6c68d13753bbdfa83516b26b6

@martinwicke I think you reviewed this CL - the author doens't have a github account.   Any idea why this is not visible in the tf.image module?

**Comment by martinwicke** [comment]
It really should be.

**Comment by prb12** [comment]
I reproduced the problem with a 0.11 binary install.  Perhaps this is fixed at head though....?

**Comment by martinwicke** [comment]
@skye, can you take a look?

**Comment by skye** [comment]
Yup. 3adb3fd is my commit btw, I'm looking into getting my github account associated with my commits.

**Comment by skye** [comment]
It looks like 3adb3fd is not included in 0.11, so the function is called `per_image_whitening`, not `per_image_standardization`. @hamidb, does the following work for you?
```
import tensorflow as tf
tf.image.per_image_whitening(...)
```

**Comment by hamidb** [comment]
@skye `per_image_whitening` works for me.
